TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -8,10 +8,49 @@
 
 import UIKit
 
-class OrganizationInfoViewController: UIViewController
+@objc class OrganizationInfoViewController: UIViewController
 {
     var organization: FFOrganization!
+    
+    let kEmployeesOrderHasChanged:String = "employeesOrderHasChanged"
+    
+    @IBAction func onRandomizeOrder()
+    {
+        var  countOrderNumbers = [Int]()
+        
+        for count in 1...self.organization.employees!.count
+        {
+            countOrderNumbers.append(count)
+        }
+        
+        var employees = self.organization.sortedEmployees!
+        
+        let countEmployees = self.organization.employees!.count
 
+        while countOrderNumbers.count > 0
```

Comment:
Why do you need so countOrderNumbers array? You should do it just with employees array.